src/constants.ts

```diff
@@ -108,10 +110,22 @@ export const viewOutput: string = localize('viewOutput', 'View Output');
 export const previewDescription: string = localize('preview', '(Preview)');
 export const pythonNewModelPreview: string = localize('pythonNewModelPreview', 'Python (Programming Model V2)');
 
+/**
+ * The "current" Node.js model is 3 (and assumed, if the number is omitted).
+ * The new Node.js model is 4.
+ * Any significantly changed new model should use an incremented number.
+ */
+export const nodejsNewModelVersion: number = 4;
+export const nodejsNewModelLabel: string = localize('nodejsNewModelPreview', 'Programming Model V4 (Preview)')
```

Comment:
<img width="448" alt="Screen Shot 2022-12-01 at 4 02 22 PM" src="https://user-images.githubusercontent.com/11282622/205184830-249e6d51-29a7-4abd-9f10-93e9b5a12c4a.png">

A few suggestions:
1. Can we remove "Programming Model" from both picks? I just don't like the duplicated stuff
2. I'd remove "(Default)" from v3. I think it looks too similar to "(Preview)" and I think "preview" is more important because people can deduce "v3" is the default since it's at the top
3. Can you add `https://aka.ms/AzFuncNodeV4` as a learn more link

Reply:
I assume that you want an actual "learn more" quick pick item appended to the bottom, but just in case, did you mean/want this little ? button?

![image](https://user-images.githubusercontent.com/5290572/205752731-46c2ab06-66a8-40e7-9be8-97f0518e3686.png)

Reply:
Yeah I wanted a real quick pick. Not sure i'd ever see that little butterino...